config/config.go

```diff
@@ -137,7 +145,7 @@ type Config struct {
 	DelayCleanTableLock uint64      `toml:"delay-clean-table-lock" json:"delay-clean-table-lock"`
 	SplitRegionMaxNum   uint64      `toml:"split-region-max-num" json:"split-region-max-num"`
 	StmtSummary         StmtSummary `toml:"stmt-summary" json:"stmt-summary"`
-	TopSQL              TopSQL      `toml:"top-sql" json:"top-sql"`
+	TopSQL              TopSQL      `toml:"-" json:"-"`
```

Comment:
Can you explain why the `toml` and `json` names are blanked out - is the goal to no longer specify them in toml, but instead use them in sysvars?

Reply:
Because the TopSQL feature doesn't want to provide a config item in the TiDB config file, only provide the `instance/global` variable to control.

Reply:
@breeswish PTAL

Reply:
Because I think it is hard for the user to understand  in this situation:

The user changes the config item in config file and restart the TiDB server, then which value takes effect? the config value or the global variable value? It will confuse the user.

Reply:
As an instance config that is not placed in the config file, is it necessary to be inside the `config.go`?

Reply:
Just for coding convenience. Of course, we can use a unique global variable to do it. Should we ?

Reply:
I will add a new global variable to store the `TopSQL` config, and remove from `config pkg`.